typing_extensions/src_py3/test_typing_extensions.py

```diff
@@ -15,9 +15,29 @@
 import typing_extensions
 import collections.abc as collections_abc
 
-TYPING_V2 = sys.version_info >= (3, 5, 1)
-TYPING_V3 = sys.version_info >= (3, 5, 3)
-TYPING_V4 = sys.version_info >= (3, 6, 1)
+TYPING_LATEST = sys.version_info[:3] < (3, 5, 0)
+TYPING_V2 = TYPING_LATEST or sys.version_info[:3] >= (3, 5, 1)
```

Comment:
I think this would be clearer as `TYPING_3_5_1` or similar, because "v2" is not a terminology used elsewhere.